### PR TITLE
Fix getting orientation for splash screen

### PR DESCRIPTION
**Pull request by tiwanek**
Splash screen orientation should not be based on natural orientation.

### CODE REVIEW

**Comment by tiwanek** [comment]
@myroot 
ping

File: runtime/browser/native_window.cc [hunk 1]
```diff
@@ -204,33 +222,32 @@ void NativeWindow::RemoveRotationHandler(int id) {
   handler_table_.erase(id);
 }
 
+NativeWindow::ScreenOrientation NativeWindow::RotationOrientation() const {
```
Comment (myroot) [inline runtime/browser/native_window.cc]: Please split the function and change the name
NativeWindow::ScreenOrientation orientation() const {
    return NativeAngleToOrientation(rotation_, natural_orientation_);
}

**static** NativeWindow::ScreenOrientation NativeAngleToOrientation(int angle, ScreenOrientation natural_orientation) {
}

Comment (tiwanek) [inline runtime/browser/native_window.cc]: ok

**Comment by tiwanek** [comment]
Please, review and integrate.

File: runtime/browser/native_window.cc [hunk 2]
```diff
@@ -20,15 +20,50 @@
...
+          kPortraitNaturalAngle :
+          kLandscapeNaturalAngle;
+  unsigned index = ARRAYSIZE(convert_table) - 1;
+  for (unsigned i = 0; i < ARRAYSIZE(convert_table); ++i) {
+    if (convert_table[i] == angle) {
+      index = i;
+      break;
+    }
+  }
+  return static_cast<NativeWindow::ScreenOrientation>(index);
```
Comment (myroot) [inline runtime/browser/native_window.cc:64]: If can't found on convert_table, index will be 6
it is out of ScreenOrientation enum.

Comment (tiwanek) [reply runtime/browser/native_window.cc:64]: I think index will be 5 (assigned in line 57).

Comment (myroot) [reply runtime/browser/native_window.cc:64]: Yes, you are right, my mistake

**Comment by myroot** [comment]
LGTM,
Sorry for too late review.

**Pull request merged**